docs/streams/developer-guide/config-streams.html

```diff
@@ -91,6 +91,7 @@
               <li><a class="reference internal" href="#rocksdb-config-setter" id="id20">rocksdb.config.setter</a></li>
               <li><a class="reference internal" href="#state-dir" id="id14">state.dir</a></li>
               <li><a class="reference internal" href="#topology-optimization" id="id31">topology.optimization</a></li>
+              <li><a class="reference internal" href="#windowed-inner-class-serde" id="id38">windowed.inner.class.serde</a></li>
```

Comment:
As pointed out on the other PR. I think we should not add this config to the docs, but rather deprecate it similar to `window.size` via the KIP you started.